usagov/usagov-2021
# USAGOV-1039 Log CEO Errors to GTM

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1039

## Description

This change pushes data to the datalayer any time we show the user an error message. This allow us to keep track of the errors that our users are getting.

## How to Test
Visit http://localhost/elected-officials-results
In your browser console, type datalayer and hit enter 
You should see a javascript object that contains this data: `error detail:  "Failed to parse address"`
Screenshot:
![Datalayer DEO API Error](https://github.com/usagov/usagov-2021/assets/1021355/7569c208-24c1-404e-9f26-85ba0d40d343)



There are other types of errors but I dont know how to make them happen.


## Checklist for the Developer



## Checklist for the Peer Reviewers
